features/step_definitions/user_steps.rb

```diff
@@ -111,6 +111,14 @@
   step(%{I am logged in as "#{login}" with password "#{DEFAULT_PASSWORD}"})
 end
 
+Given "I am logged in as a new user {string}" do |login|
+  step(%{I am logged in as "#{login}"})
+  user = User.find_by(login: login)
+  user.created_at = Time.now.utc
+  user.confirmed_at = Time.now.utc
+  user.save
```

Comment:
```suggestion
  user.save!
```
It'll throw if something is invalid.